modules/tinymce/src/core/main/ts/api/html/Styles.ts

```diff
@@ -268,13 +269,15 @@ const Styles = (settings: StylesSettings = {}, schema?: Schema): Styles => {
           }
         }
         // Compress the styles to reduce it's size for example IE will expand styles
-        compress('border', '', true);
-        compress('border', '-width');
-        compress('border', '-color');
-        compress('border', '-style');
-        compress('padding', '');
-        compress('margin', '');
-        compress2('border', 'border-width', 'border-style', 'border-color');
+        if (compressed) {
```

Comment:
```suggestion
        if (shouldCompress) {
```
Again, compressed implies already compressed, so rename for clarity.